enqueue/enqueue-bundle/0.3/etc/packages/test/enqueue.yaml

```diff
@@ -0,0 +1,6 @@
+enqueue:
+    transport:
+        default: 'null'
+        'null': ~
```

Comment:
it defines the configuration for the NullTransport. But the key must be quoted in YAML, as unquoting it would parse it as `null`, not as the string `'null'`